src/components/apply/avatar.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Image from "next/image";
+
+type AvatarType = {
+  avatar:
+    | "Wildlife Wanderer"
+    | "City Cruiser"
+    | "Foodie Fanatic"
+    | "Beach Bum"
+    | null
+    | undefined;
+};
+
+const Avatar = ({ avatar }: AvatarType) => {
+  const getAvatarImage = () => {
+    switch (avatar) {
+      case "Wildlife Wanderer":
+        return "/images/wildlifewanderer.svg";
+      case "City Cruiser":
+        return "/images/citycruiser.svg";
+      case "Foodie Fanatic":
+        return "/images/foodiefanatic.svg";
+      case "Beach Bum":
+        return "/images/beachbum.svg";
+      default:
+        return "";
+    }
+  };
```

Comment:
can we move this function out of the component? it can take in avatar: AvatarType.